packages/react-components/lib/tasks/types/clean.tsx

```diff
@@ -0,0 +1,123 @@
+import { Autocomplete, TextField } from '@mui/material';
+import React from 'react';
+
+interface GoToPlaceActivity {
+  category: string;
+  description: string;
+}
+
+interface CleanActivity {
+  category: string;
+  description: {
+    unix_millis_action_duration_estimate: number;
+    category: string;
+    expected_finish_location: string;
+    description: {
+      zone: string;
+    };
+    use_tool_sink: boolean;
+  };
+}
+
+export interface CleanTaskDescription {
+  category: string;
+  phases: [
+    cleanPhase: {
+      activity: {
+        category: string;
+        description: {
+          activities: [goToPlaceActivity: GoToPlaceActivity, cleanActivity: CleanActivity];
+        };
+      };
+    },
+  ];
+}
```

Comment:
That is outdated now, because we are now using `perform_action` to do a cleaning action, https://github.com/open-rmf/rmf_demos/blob/main/rmf_demos_tasks/rmf_demos_tasks/dispatch_clean.py

Reply:
You mean the schemas are outdated? Can you open a PR to fix it in rmf?

Reply:
No the schemas are not outdated, just it is no longer used in rmf_demos. I believe there were past deployments that were using the schemas, so we should not be changing anything on that end

Reply:
iiuc, we have a native "clean" task and a custom "clean" task that is actually a compose task that includes the native "clean"? This presents a problem with the way we define `supportedTasks`. The name `clean` now refers to the compose-clean, and since the "name" is also used to determine the category for non-compose task, we end up with no way to define both a native clean and compose-clean task.

This would be solved if we use "name" and "type" to define `supportedTasks`, the compose-clean task could be defined as `{ "name": "clean", "type": "composeClean" }`.